recipes/awslambdaric/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "awslambdaric" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/awslambdaric-{{ version }}.tar.gz
+  sha256: dad646f566aa7ec9b7179f16ca6741a2bea148abec6ed5947f86d00607e0a9a2
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - importlib-metadata >=2.0  # [py<38],==2.*
+    - python
+    - simplejson ==3.17.2
```

We need to relax the simplejson dependency because version 3.17.2 was only built against python 3.6.

```suggestion
    - simplejson >=3.17.2,<3.18
```